sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -94,15 +122,22 @@ class RetentionPolicy(GeneratedRetentionPolicy):
         soft-deleted data should be retained. All data older than this value will
         be deleted.
     """
+    
+    enabled: bool = False
+    """Indicates whether a retention policy is enabled for the storage service."""
+    days: int = None
+    """Indicates the number of days that metrics or logging or
+        soft-deleted data should be retained. All data older than this value will
+        be deleted."""
```

Comment:
Do we want these comments to indent? I have it like this everywhere (was how VSCode formatted it), but I couldn't find any good online examples to check